test/unit/test-helpers/CassandraDataInserter.js

```diff
@@ -33,24 +33,35 @@ module.exports = class CassandraDataInserter {
         })
     }
 
-    bulkInsert(n) {
+    bulkInsert(nbNotSigned, nbSigned) {
```

Comment:
Continuing on the same theme: Does signed vs. unsigned make any difference from the point of view of the code being tested (it might! I'm not 100% sure where this class is being used)?

For example, `CassandraUtil.test.js` uses this inserter but `CassandraUtil.js` contains no changes related to the signatures!

So I guess you could always insert signed data (because why not)?